public/assets/js/scripts.js

```diff
@@ -285,4 +294,131 @@ $(document).ready(function () {
     $(this).html() == '<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>' ? $(this).html('<span><img src="./assets/img/icons/view-map-pin.svg" alt="view map"> &nbsp; View Map</span>') : $(this).html('<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>');
   });
 
+  $('.search-calculate-button').click(function(){
+    const periodic = $('.periodic-investment').val();
+    const initial = $('.initial-investment').val();
+    const avgPropertyCost = removeCommasAndMakeNumber($('.investment-value-input').data('max-amount'));
+    const frequency = $("input[name='investment-frequency']:checked").val();
+
+    const balance = avgPropertyCost - initial;
+    let output = [];
+    
+    let popOverIcon = `<a tabindex="0" class="" id="search-ready-awesome-spread" role="button" data-toggle="popover">
+                        <img src="./assets/img/icons/question-mark.svg" alt="payment">
+                      </a>`;
+    let recommendation = '';
+    let recommendationNav = '';
+    let recommendationBody = '';
+    
+    // Recommendations breakdown can be found here - https://docs.google.com/document/d/1gsomOY9qclUz9RzadN3ztJH4Y0ryGT-fukNBFLhJAIU/edit?pli=1#heading=h.yy9lcow7gkem
+    
+    const outrightPersonal = initial >= (avgPropertyCost * 0.5) && balance / periodic < 6 / frequency;
+    const immediatePrivate = initial >= (avgPropertyCost * 0.25) && (avgPropertyCost <= 45000000);
+    const outrightMortgage = initial >= (avgPropertyCost * 0.25);
+    const directSpread = initial >= (avgPropertyCost * 0.2) && balance / periodic > 6 / frequency && balance / periodic <= 24 / frequency;
+    const immediateFederal = initial >= (avgPropertyCost * 0.1) && (avgPropertyCost <= 15000000) && balance / periodic <= 12 / frequency;
+    const spreadFederal = initial >= (avgPropertyCost * 0.01) && (avgPropertyCost <= 15000000) && balance / periodic <= 24 / frequency;
+    const rentToOwn = initial >= (avgPropertyCost * 0.05) && balance / periodic <= 120 / frequency;
+    const spreadPrivate = initial >= (avgPropertyCost * 0.025) && (avgPropertyCost <= 45000000) && balance / periodic <= 24 / frequency;
+    const hybrid = initial >= (avgPropertyCost * 0.01) && balance / periodic <= 24 / frequency;
+    
+
+    if (outrightPersonal || outrightMortgage) {
```

Comment:
Let's define an object that caters for all the categories and assign them here.
```suggestion
    const categories = {
      outright: {
         title: 'Outright',
         percentage: '50%'
      },
      mortgage: {
         title: 'Mortgage',
         percentage: '25%'
      },
      ...
    }
    if (outrightPersonal || outrightMortgage) {
       output.push(categories.outright);
    }
    ....
```

Having it in a single object makes it easier to update the values of each categories without going through the code. Also, the image tag in the title and the message are constant, so they can be used in the final implementation (when you are building the output)